### PR TITLE
Feature/cyprus

### PR DESCRIPTION
**Pull request by ultibo**
First draft for cyprus parser and config. first pull request on that project, please let me know if some changes are needed.

### CODE REVIEW

**Comment by ultibo** [comment]
Ok sure i'll modify it to return every time all the data points for the day and not just the last one

**Comment by ultibo** [comment]
Ok i did something that is supposed to work, let me know if some other changes are needed. I wasn't able to test end-to-end, only by running the parser locally.

File: parsers/CY.py [hunk 1]
```diff
@@ -0,0 +1,186 @@
...
+                'solar': production[time].get('solar', 0.0),
+                'wind': production[time].get('wind', 0.0),
+                # as discussed in issue 122 on github, put all the non wind/solar as unknown.
+                # it will be assigned a "fossil fuel" co2 emission intensity.
+                # also, as we only have the total production, we should deduce solar and wind
+                'unknown': production[time].get('total', 0.0) - production[time].get('solar', 0.0) - production[time].get('wind', 0.0)
+            },
+            'storage': {},
+            'source': 'http://www.dsm.org.cy',
+            'datetime': time,
```
Comment (corradio) [inline parsers/CY.py]: you need to return a python datetime object, i.e. `time.datetime`.

File: parsers/CY.py [hunk 2]
```diff
@@ -0,0 +1,186 @@
...
+            'production': {
+                'solar': production[time].get('solar', 0.0),
+                'wind': production[time].get('wind', 0.0),
+                # as discussed in issue 122 on github, put all the non wind/solar as unknown.
+                # it will be assigned a "fossil fuel" co2 emission intensity.
+                # also, as we only have the total production, we should deduce solar and wind
+                'unknown': production[time].get('total', 0.0) - production[time].get('solar', 0.0) - production[time].get('wind', 0.0)
+            },
+            'storage': {},
+            'source': 'http://www.dsm.org.cy',
```
Comment (corradio) [inline parsers/CY.py]: Can you use `dsm.org.cy`? The convention is to just show the domain name.

**Pull request merged**